### ISSUE TITLE
Ban Appeal

### ISSUE THREAD

**Issue by xxmtgxx** [ObliqueNET/Server#1056]
I recently got banned, and I know what I did was wrong. I promise to never do it again. I would just like to keep playing on your server. IGN:xxmtgxx

**Comment by Rasgnarok** [comment]
You were banned for doctoring images in our # support Discord channel. You tried abusing volunteered, free refunds that people genuinely need when they **ACTUALLY** lose Pokémon to Pixelmon's numerous battle bugs. Players often ask for refunds with screenshots, and you doctored (extremely poorly) an image to get yourself a shiny Pokémon you never had actually found, playing stupid when confronted. 

In short, you do not get to recover and file a ban appeal on this one. It is because of people like you, that players are questioned before any refunds - and your attempts at editing could even ruin their chance at refunds at all.

Your appeal is denied, your ban is permanent.
